install/kubernetes/istio-auth.yaml

```diff
@@ -218,18 +207,12 @@ spec:
         - mountPath: /etc/certs
           name: istio-certs
           readOnly: true
-        - mountPath: /etc/istio/ingress-certs
```

Comment:
we can no longer remove these since pilot PR got merged to require these

Reply:
well the template needs to be fixed... any change (including the bot auto update) will erase these until the template is fixed

Reply:
yes, you cannot remove the imgress secrets.

Reply:
Which template do we use to generate these files?

Reply:
the one mentioned at the beginning of the file

Reply:
https://github.com/istio/istio/blob/master/install/updateVersion.sh#L155

Reply:
I will fix it in a separate PR.